RMv4_Recipely/RMv4_Recipely/User Info/Profile Module/ProfileViewController.swift

```diff
@@ -0,0 +1,175 @@
+
+    private enum Constants {
+        static let navigationTitleText = "Profile"
+        static let verdanaBoldFontName = "Verdana-Bold"
+        static let headerTableViewCellIdentifier = "HeaderTableViewCell"
+        static let optionsTableViewCellIdentifier = "OptionsTableViewCell"
+    }
+
+    // MARK: - Visual Components
+
+    private let tableView = UITableView()
+
+    private let profileTitleBarButtonItem: UIBarButtonItem = {
+        let label = UILabel()
+        label.text = Constants.navigationTitleText
+        label.font = UIFont(name: Constants.verdanaBoldFontName, size: 28)
+        let item = UIBarButtonItem(customView: label)
+        return item
+    }()
+
+    // MARK: - Public Properties
+
+    var presenter: ProfilePresenterProtocol?
+
+    // MARK: - Private Properties
+
+    private let optionCells: [OptionCell] = [
+        OptionCell(type: .bonuses, title: "Bonuses", imageName: "star.fill"),
+        OptionCell(type: .terms, title: "Terms & Privacy Policy", imageName: "doc.fill"),
+        OptionCell(type: .logOut, title: "Log Out", imageName: "rectangle.portrait.and.arrow.right.fill")
+    ]
+
+    private let tableViewSections: [ProfileTableViewCell] = [
+        .headerCell,
+        .cells
+    ]
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+
+    // MARK: - Private Methods
+
+    private func setupUI() {
+        view.backgroundColor = .white
+        navigationItem.leftBarButtonItem = profileTitleBarButtonItem
+        setupTableView()
+    }
+
+    private func setupTableView() {
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.separatorStyle = .none
+
+        tableView.register(HeaderTableViewCell.self, forCellReuseIdentifier: Constants.headerTableViewCellIdentifier)
+        tableView.register(OptionsTableViewCell.self, forCellReuseIdentifier: Constants.optionsTableViewCellIdentifier)
+
+        view.addSubview(tableView)
+        tableView.dataSource = self
+        tableView.delegate = self
+
+        setTableViewConstraints()
+    }
+
+    private func setTableViewConstraints() {
+        tableView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor).isActive = true
+        tableView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor).isActive = true
+        tableView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor).isActive = true
+        tableView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor).isActive = true
+    }
+}
+
+/// Расширение вью методами протокола-интерфейса ProfileViewControllerProtocol
+extension ProfileViewController: ProfileViewControllerProtocol {
+    func reloadTableView() {
+        tableView.reloadData()
+    }
+
+    func showNameEditorAlert() {
+        let alert = UIAlertController(title: "Change your name and surname", message: nil, preferredStyle: .alert)
```

Comment:
```suggestion
        let alert = UIAlertController(title: "", message: nil, preferredStyle: .alert)
```
Константы либо присылать их презентера